### PR TITLE
feat: convert path alias into relative path on CI/CD

### PR DESCRIPTION
**Pull request by devRickyyy**
1. Using babel convert the "@/*" alias into the relative path. 
2. Modify the configurations of semantic-release for publishing the converted sources.

### CODE REVIEW

**Comment by hsluoyz** [comment]
@leo220yuyaodog @wenxuan70 plz review

File: package.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
   "license": "Apache-2.0",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
+    "build": "babel src --out-dir dist/src && babel index.js --out-dir dist/ && cp package.json dist/ && cp LICENSE dist/",
```
Comment (wenxuan70) [inline package.json:9]: missing `@babel/cli` dependency.

Comment (devRickyyy) [reply package.json:9]: Thanks for help! I have added the dependency and test it with `semantic-release --dry-run`

File: package.json [hunk 2]
```diff
@@ -6,13 +6,13 @@
   "license": "Apache-2.0",
```
Comment (hsluoyz) [inline package.json:6]: Don't add [package-lock.json](https://github.com/casdoor/casdoor-vue-sdk/pull/13/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519)

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Comment by hsluoyz** [comment]
@leo220yuyaodog

**Comment by leo220yuyaodog** [comment]
The pr works well in my localhost. 
image

**Pull request merged**